openedx/features/survey_report/management/commands/generate_report.py

```diff
@@ -0,0 +1,31 @@
+"""
+CLI command to generate survey report.
+"""
+
+from django.core.management.base import BaseCommand, CommandError
+
+from openedx.features.survey_report.api import generate_report
+
+
+class Command(BaseCommand):
+    """
+    Management command to generate a new survey report with
+    non-sensitive data.
+    """
+
+    help = """
+        This command will create a new survey report using some
+        models to get:
+        - Total number of courses offered
+        - Currently active learners
+        ...
```

Comment:
```suggestion
       - Learners ever registered
       - Generated certificates
```
In my previous comment, I meant listing the reports being generated